test/controllers/SignOutControllerSpec.scala

```diff
@@ -47,4 +48,15 @@ class SignOutControllerSpec extends ItSpec {
     }
   }
 
+  "exitSurveyPaye should" - {
+    "redirect to feedback frontend with eSSTTP-PAYE as the service identifier" in {
+      AuthStub.authorise()
+      EssttpBackend.SubmitArrangement.findJourney()
+      val fakeRequest = FakeRequest().withAuthToken().withSession(SessionKeys.sessionId -> "IamATestSessionId")
+      val result: Future[Result] = controller.exitSurveyPaye(fakeRequest)
+      status(result) shouldBe Status.SEE_OTHER
+      redirectLocation(result) shouldBe Some("http://localhost:9514/feedback/eSSTTP-PAYE")
```

Comment:
you could check the session is empty as well here - I think you can do something like `session(result)` of something to gain access to it